storage_backend_s3/components/s3_adapter.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2019 Camptocamp SA (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import io
+import logging
+
+from odoo import exceptions
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import boto3
+    from botocore.exceptions import ClientError, EndpointConnectionError
+
+except ImportError as err:  # pragma: no cover
+    _logger.debug(err)
```

Comment:
Since it's already declared in the manifest's `external_dependencies`, we can drop the try except block here